app/frontend/src/components/RequestForm.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <v-container class="request-form">
+    <h2 class="text-center mb-10">Request Account</h2>
+
+    <p>Please submit the Acronym of the application you wish to add. You will get an email once it is confirmed.</p>
+
+    <v-form ref="form" v-model="valid" lazy-validation>
+      <v-row>
+        <v-col>
+          <label>IDIR</label>
+          <v-text-field
+            readonly
+            hide-details="auto"
+            solo
+            single-line
+            dense
+            outlined
+            flat
+            :value="userInfo.idir"
+          ></v-text-field>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col>
+          <label>E-mail</label>
+          <v-text-field
+            readonly
+            hide-details="auto"
+            solo
+            dense
+            single-line
+            outlined
+            flat
+            :value="userInfo.emailAddress"
```

Comment:
Wondering if this should be bound as a v-model to the respective values instead of just value - even if the fields are read-only.

Reply:
yeah sounds like a good idea